### PR TITLE
AB-3255: Added Update Government Access Flow -Part 1 

### PR DESCRIPTION
**Pull request by carolinegeorge-gov**
### Description

Initial set up for the newly added Update Government Access flow

### Related Azure Boards Work Items

[AB#3255](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3255)

### Checklist

- [X] I have tested the changes locally
- [X] I have updated the documentation if necessary
- [X] I have added/updated tests that prove my fix is effective or that my feature works
- [X] I have checked that my code follows the project's coding style by running `npm run format:check`
- [X] I have checked that my code contains no linting errors by running `npm run lint`
- [X] I have checked that my code contains no type errors by running `npm run typecheck`
- [X] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [X] I have checked that all e2e tests pass by running `npm run test:e2e`

### CODE REVIEW

File: frontend/public/locales/en/access-to-governmental-benefits.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        "back": "Back"
+      }
+    },
+    "view": {
+      "page-title": "Access to governmental benefits",
+      "intro-text": "You have indicated that you have access to the following governmental benefits.",
+      "provincial-or-territorial-dental-benefit": "Provincial or Territorial dental benefit:",
+      "alberta-child-health-benefits": "Alberta Child Health Benefits",
+      "federal-benefits-dental-benefits": "Federal benefits dental benefits:",
+      "dental-service-program": "Dental  Service Program",
```
Comment (nicholas-ly) [inline frontend/public/locales/en/access-to-governmental-benefits.json:18]: "Alberta Child Health Benefits" and "Dental Services Program" (and their French names) should be coming from the lookup service (`getAllProvincialTerritorialSocialPrograms` and `getAllFederalSocialPrograms`) - no need to define i18n keys for them.

File: frontend/public/locales/en/access-to-governmental-benefits.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+
+{
+  "access-to-governmental-benefits": {
+    "confirm": {
+      "page-title": "Application submitted",
+      "alert-heading": "You have successfully applied for the Canadian Dental Care Plan!",
+      "your-info-has-been-updated": "Your information has been updated.",
+      "button": {
+        "back": "Back"
+      }
```
Comment (nicholas-ly) [inline frontend/public/locales/en/access-to-governmental-benefits.json:10]: Can we remove all this text for `"confirm"`? I don't see them in the Figma.

File: frontend/public/locales/en/access-to-governmental-benefits.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        "option-no": "<strong>No</strong>, I do not have <strong>provincial or territorial</strong> benefits",
+        "social-programs": {
+          "input-legend": "If yes, through which province or territory?",
+          "radio-legend": "Please select which social program you are covered under. If you have more than one, please select the one you use most."
+        }
+      },
+      "button": {
+        "back": "Back",
+        "continue": "Continue",
+        "cancel-btn": "Cancel",
```
Comment (nicholas-ly) [inline frontend/public/locales/en/access-to-governmental-benefits.json:60]: Can we remove "Continue" and "Cancel"? I don't see them in the Figma.

File: gitops/overlays/dev/configs/frontend/config.conf [hunk 4]
```diff
@@ -11,7 +11,7 @@ OTEL_TRACES_ENDPOINT=https://dynatrace.dev-dp.admin.dts-stn.com/e/21a07aef-852b-
 #
 # Application feature flags configuration
 #
-ENABLED_FEATURES=doc-upload,email-alerts,hcaptcha,view-personal-info,view-applications,view-letters,view-messages,edit-personal-info,status,authenticated-status-check,show-prototype-banner
+ENABLED_FEATURES=doc-upload,email-alerts,hcaptcha,view-personal-info,view-applications,view-letters,view-messages,edit-personal-info,status,authenticated-status-check,show-prototype-banner, update-governmental-benefit
```
Comment (nicholas-ly) [inline gitops/overlays/dev/configs/frontend/config.conf:14]: @gregory-j-baker told me to create new PRs for changes to the gitops sub-project so I will pay it forward.

File: frontend/public/locales/en/access-to-governmental-benefits.json (hunk 1 above)
Comment (carolinegeorge-gov) [reply frontend/public/locales/en/access-to-governmental-benefits.json]: Removed: 3e3028732d9cee2c8c61af92f40a0aa376b10515.

File: frontend/public/locales/en/access-to-governmental-benefits.json (hunk 3 above)
Comment (carolinegeorge-gov) [reply frontend/public/locales/en/access-to-governmental-benefits.json]: Removed: 3e3028732d9cee2c8c61af92f40a0aa376b10515.

File: gitops/overlays/dev/configs/frontend/config.conf (hunk 4 above)
Comment (carolinegeorge-gov) [reply gitops/overlays/dev/configs/frontend/config.conf:14]: Removed: 3e3028732d9cee2c8c61af92f40a0aa376b10515.

**Review by nicholas-ly — DISMISSED** [review]

**Review by nicholas-ly — APPROVED** [review]

**Pull request merged**